### PR TITLE
Update local ssd examples to use local ssd startup solution

### PR DESCRIPTION
**Pull request by alyssa-sm**
This PR updates the local ssd examples to use the RAID solution outlined in #2720.

#### Manual Test
The following test was successfully completed for both blueprints.
- Deployed blueprint
- Submitted a job 
- Verified that the RAID was successful on the nodes.

### CODE REVIEW

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml [hunk 1]
```diff
@@ -37,6 +37,14 @@ deployment_groups:
     settings:
       local_mount: /home
 
+  - id: startup
+    source: ./modules/scripts/startup-script
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd-v5-legacy.yaml:41]:
```suggestion
    source: modules/scripts/startup-script
```

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml [hunk 2]
```diff
@@ -37,6 +37,14 @@ deployment_groups:
...
+    settings:
+      local_ssd_filesystem:
+        fs_type: "ext4"
+        mountpoint: "/mnt/localssd"
+        permissions: "0755"
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd-v5-legacy.yaml:46]: I think it's worth being explicit to users about the string format for the filesystem permissions. Without the quotes, it gets parsed to the number 493. Removing the quotes on strings is a `nit`, but I prefer the quoteless style where possible as it makes YAML more human legible than machine-readable JSON.

```suggestion
        fs_type: ext4
        mountpoint: /mnt/localssd
        permissions: "0755" # must quote numeric filesystem permissions!
```

File: community/examples/hpc-slurm-local-ssd.yaml [hunk 3]
```diff
@@ -37,9 +37,17 @@ deployment_groups:
     settings:
       local_mount: /home
 
+  - id: startup
+    source: ./modules/scripts/startup-script
+    settings:
+      local_ssd_filesystem:
+        fs_type: "ext4"
+        mountpoint: "/mnt/localssd"
+        permissions: "0755"
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd.yaml:46]:
```suggestion
        fs_type: ext4
        mountpoint: /mnt/localssd
        permissions: "0755" # must quote numeric filesystem permissions!
```

File: community/examples/hpc-slurm-local-ssd.yaml [hunk 4]
```diff
@@ -37,9 +37,17 @@ deployment_groups:
     settings:
       local_mount: /home
 
+  - id: startup
+    source: ./modules/scripts/startup-script
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd.yaml:41]:
```suggestion
    source: modules/scripts/startup-script
```

**Review by tpdownes — CHANGES_REQUESTED** [review]

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml [hunk 5]
```diff
@@ -37,6 +37,14 @@ deployment_groups:
...
+
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd-v5-legacy.yaml:47]: Can you please add a comment here identical to #2871 about the loss of data when shutting nodes down without specifying local SSD data migration?

File: community/examples/hpc-slurm-local-ssd.yaml [hunk 6]
```diff
@@ -37,9 +37,17 @@ deployment_groups:
...
+    settings:
+      local_ssd_filesystem:
+        fs_type: "ext4"
+        mountpoint: "/mnt/localssd"
+        permissions: "0755"
+
```
Comment (tpdownes) [inline community/examples/hpc-slurm-local-ssd.yaml:47]: Same comment (about data migration) here, please.

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml (hunk 1 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd-v5-legacy.yaml]: Done

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml (hunk 2 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd-v5-legacy.yaml]: Done

File: community/examples/hpc-slurm-local-ssd.yaml (hunk 4 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd.yaml]: Done

File: community/examples/hpc-slurm-local-ssd.yaml (hunk 3 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd.yaml]: Added!

File: community/examples/hpc-slurm-local-ssd-v5-legacy.yaml (hunk 5 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd-v5-legacy.yaml:52]: Added!

File: community/examples/hpc-slurm-local-ssd.yaml (hunk 6 above)
Comment (alyssa-sm) [reply community/examples/hpc-slurm-local-ssd.yaml:52]: Added!

**Review by tpdownes — APPROVED** [review]

**Pull request merged**